### ISSUE TITLE
New Site Construction

### ISSUE THREAD

**Issue by ToxicBakery** [Rajawali/Rajawali#766]
Alright, a design was picked. Now for the fun. I have included the requirements posted from the original thread. We now need to figure out how we are going to throw this together. The first step is to get the original PSD/PNG from Andrew so that we can break it up appropriately.

I know we have a lot of web designers here including myself that are more than capable of doing this on their own. For the spirit of it being open source though I would like everything to be allocated in tasks

## Tasks
- [x] **Toxic Bakery** -  Grab the originals from Andrew.
- [x] **MasDennis** - Create a repository for the site.
- [ ] **???** - Convert the mockup to a basic template.
- [ ] **???** - Create the template for each page.
- [ ] **???** - Content content content

## Requirements
* A list of features, particularly those that make Rajawali stand out. For me being 4.0+ and no NDK makes Rajawali immensely attractive now.  
* A general breakdown of how the rendering process in Rajawali is intended to work
* Host the wiki? Or should this stay put?

*Overall*
- A simple, uncluttered layout. Responsive (mobile/desktop friendly).
- Small Rajawali logo (top left)
- Menu at the top (top right) (Gallery, Tutorials, Documentation, Issues, Team)
- Light colours
- Google AdSense integration

*Homepage*
- Short description of the framework with some of the key features (4.0+, no NDK, ES 2.0, etc)
- 6 or 8 square images of the most beautiful Rajawali apps/wallpapers. With a link to the complete gallery.
- "Start building 3D content now" with a link to Github (tags & unstable version w/ disclaimer)
- Icons w/ short description + links to same items as menu (Gallery, Tutorials, Documentation, Issues)

*Gallery*
- Square tiles with links to Google Play. Clean, simple layout

*Documentation*
- JavaDoc but sexier than it currently is, well integrated with the site

*Issues*
- Basically just links to the Github Issues page

*Team*
- Team profiles. Avatar, short description, role + social media links.

Mods, feel free to edit this post as needed to add clarity or update as progress is made.

**Comment by dcerisano** [comment]
I found a bootstrap that fits your requirements an posted it on your google+ thread.

Unfortunately, I stumbled upon a @jwoolston trigger word (which shall not be mentioned again) and things got a little weird.

So the question is, could you get this into a repo so we can have a go at it?

https://github.com/IronSummitMedia/startbootstrap-business-casual

**Comment by ToxicBakery** [comment]
Yeah sure, is it alright if I just make a blank repo and let you drop in whatever you need?

**Comment by ToxicBakery** [comment]
It is worth noting that Dennis has people working on this so you may want to collaborate via the issues on that repo here forward.
